crates/symbolicli/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "symbolicli"
+version = "0.1.0"
```

Comment:
Won't craft override it as `0.6.0` anyway to keep it in sync with other crates in the monorepo?

Reply:
Yeah, you're probably right